bittide-instances/src/Bittide/Instances/Pnr/Ethernet.hs

```diff
@@ -26,11 +28,34 @@ import Bittide.ProcessingElement
 import Bittide.Wishbone
 import Protocols.Idle
 
+-- Simulation imports
```

Comment:
Maybe it's super overkill, but this sounds like `LANGUAGE CPP` could be nice for conditional imports and also later down in the file for the memory contents?

Reply:
I added two flags, one to include the binaries in the compilation named `cpu-include-binaries`, and `sim-baud-rate` that sets the baud rate to the maximum allowed by the domain for faster simulation.

Reply:
I know I'm on holiday :eyes:, but couldn't we use `clashSimulation` for this? CPP is somewhat terrible in the sense that it's very hard to keep all combinations of CPP compiling / testable.